app/screens/index.js

```diff
@@ -78,6 +78,9 @@ export function registerScreens(store, Provider) {
     Navigation.registerComponent('Settings', () => wrapper(require('app/screens/settings/general').default), () => require('app/screens/settings/general').default);
     Navigation.registerComponent('SidebarSettings', () => wrapper(require('app/screens/settings/sidebar').default), () => require('app/screens/settings/sidebar').default);
     Navigation.registerComponent('SSO', () => wrapper(require('app/screens/sso').default), () => require('app/screens/sso').default);
+    if (__DEV__) {
+        Navigation.registerComponent('StoryBook', () => wrapper(require('storybook').default), () => require('storybook').default);
```

Comment:
does this screen need access to redux and the intl provider? if not, there is no need to wrap it